data-raw/incubation-period.R

```diff
@@ -1,13 +1,14 @@
 library(data.table)
 
-covid_incubation_period <- data.table(
+incubation_periods <- data.table(
   as_reported = "5.06 (log SD 0.418)",
   mean = 1.621, mean_sd = 0.0640, 
   sd = 0.418, sd_sd = 0.0691,
   dist = "lognorm",
+  disease = "SARS-CoV-2",
```

Comment:
Typically "magic strings" are frowned upon as they are an easy way to trip up. Other languages might use Enumerations for this. The simplest suggestion I have found for R is to create a central list:

central code:
```
diseases <- list(SARS_COV_2 = "SARS-CoV-2", ...)
incubation_period_sources <- list(LAUER = "lauer", ...)
```

various use cases:
``` disease = diseases$SARS_COV_2 ```

```incubation_period <- get_incubation_period(disease = diseases$SARS_COV_2, source = incubation_period_sources$LAUER, max_value = 15)```

Any code that then needs to test can check `if disease == diseases$SARS_COV_2` avoiding any typo's slipping through unnoticed.

Reply:
A quick search suggests 28 different places where "SARS-CoV-2" is used (mostly in use cases) but it highlights the repeated use of the magic string

Reply:
note SHOUTY_SNAKE is typical for enums / constants in some other languages even when they are not enforced by the code - I can't find it covered on a style guide anywhere for R

Reply:
So I think this is just for filtering a data frame that the users will be able to inspect (that was certainly the intended implementation). Happy to consider other approaches to this but not sure how it would work in practice. 

Definitely agree in general about strings and if else checks - which is an issue with the new method arg. Suggest we discuss further in an issue.